lib/AppConfig.php

```diff
@@ -140,10 +163,28 @@ public function getAppValue($key) {
 	 * @param string $key
 	 * @param string $value
 	 *
-	 * @return string
+	 * @return void
 	 */
 	public function setAppValue($key, $value) {
-		return $this->config->setAppValue($this->appName, $key, $value);
+		$this->validateValue($key, $value);
```

Comment:
This call throws an "UnexpectedValueException", so the method should either document it, or handle that particular exception

Reply:
`throws` has been added